application/backend/app/schemas/sink.py

```diff
@@ -151,3 +153,41 @@ class WebhookSinkConfig(BaseSinkConfig):
 ]
 
 SinkAdapter: TypeAdapter[Sink] = TypeAdapter(Sink)
+
+# ---------------------------------
+# Creation Schemas (POST requests)
+# ---------------------------------
+# These schemas inherit from HasID first to override the required ID field with an auto-generated one (if absent) via
+# MRO (Method Resolution Order).
+
+
+class DisconnectedSinkConfigCreate(HasID, DisconnectedSinkConfig):
+    pass
+
+
+class FolderSinkConfigCreate(HasID, FolderSinkConfig):
+    pass
+
+
+class MqttSinkConfigCreate(HasID, MqttSinkConfig):
+    pass
+
+
+class RosSinkConfigCreate(HasID, RosSinkConfig):
+    pass
+
+
+class WebhookSinkConfigCreate(HasID, WebhookSinkConfig):
+    pass
+
+
+SinkCreate = Annotated[
+    FolderSinkConfigCreate
+    | MqttSinkConfigCreate
+    | RosSinkConfigCreate
+    | WebhookSinkConfigCreate
+    | DisconnectedSinkConfigCreate,
```

Comment:
Can you exclude the "disconnected" sink from the list of sinks that can be created?

Reply:
Good point! Removed both.